influxdb_client/client/write/dataframe_serializer.py

```diff
@@ -20,7 +20,8 @@ def _itertuples(data_frame):
 
 
 def _not_nan(x):
-    return x == x
+    from ...extras import pd
```

Comment:
I see this same import occurs on line 91.  Wouldn't it be more efficient to import this once at the head of the file?

Reply:
@karel-rehor, nice catch! Thank you 👍. I've uploaded the fixed version.